scripts/deployBridge.ts

```diff
@@ -0,0 +1,124 @@
+import {
+  StarknetContract,
+  StarknetContractFactory,
+  Account,
+} from "hardhat/types";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import fs from "fs";
+import { Contract, ContractFactory } from "ethers";
+import hre, { starknet, network, ethers } from "hardhat";
+
+/**
+ * deploys and initializes ETHStaticAToken on L2
+ * @param deployer the deployer starknet account
+ * @param proxy_admin address of the proxy owner
+ * @param governor_address address of the bridge controller/owner
+ */
+export async function deployL2Bridge(
+  deployer: Account,
+  proxy_admin: bigint,
+  governor_address: bigint
+) {
+  let proxiedBridge: StarknetContract;
+  let bridgeImplementation: StarknetContract;
+  let proxyFactoryL2: StarknetContractFactory;
+  let proxyBridge: StarknetContract;
+
+  const L2BridgeFactory = await starknet.getContractFactory("token_bridge");
+  proxyFactoryL2 = await starknet.getContractFactory("proxy");
+
+  console.log("deploying L2 proxy bridge...");
+  proxyBridge = await proxyFactoryL2.deploy({
+    proxy_admin: proxy_admin,
+  });
+
+  bridgeImplementation = await L2BridgeFactory.deploy();
+
+  await deployer.invoke(proxyBridge, "initialize_proxy", {
+    implementation_address: BigInt(bridgeImplementation.address),
+  });
+
+  fs.writeFileSync(
+    `deployment/L2Bridge.json`,
+    JSON.stringify({
+      governor_address: deployer.starknetContract.address,
```

Comment:
the value should be `governor_address` but since it's equal to `deployer.starknetContract.address`, it works. :)